### ISSUE TITLE
can´t fly 

### ISSUE THREAD

**Issue by yiyongrobotics** [DREAMS-lab/nsf_cps_challenge#1]
HI,

Thanks for your cool work, during trying the demo I found some issues below and the drone can´t fly.
![image](https://user-images.githubusercontent.com/26245593/61121419-77e89f80-a49f-11e9-9b71-5ea05de301f0.png)
bset,
Yiyong

**Comment by harishanand95** [comment]
Is the drone model visible in Gazebo?

**Comment by yiyongrobotics** [comment]
> Is the drone model visible in Gazebo?

yes, it is visible in gazebo, but it cant be set up . when l launch all commands, no any action.

**Comment by harishanand95** [comment]
Could you try the following commands and check if there are any erros/ drone is flying?

In terminal 1,
```
roslaunch nsf_cps_challenge openuav.launch
```
In terminal 2,
```
cd ~/catkin_ws/src/nsf_cps_challenge/scripts
python vs_without_yolo.py	
``` 
Do the following command after drone is visible in gazebo.
In terminal 3,
```
rosservice call /mavros/set_mode "base_mode: 0 custom_mode: 'OFFBOARD'"
rosservice call /mavros/cmd/arming "value: true"
```

**Comment by yiyongrobotics** [comment]
> Could you try the following commands and check if there are any erros/ drone is flying?
> 
> In terminal 1,
> 
> ```
> roslaunch nsf_cps_challenge openuav.launch
> ```
> 
> In terminal 2,
> 
> ```
> cd ~/catkin_ws/src/nsf_cps_challenge/scripts
> python vs_without_yolo.py	
> ```
> 
> Do the following command after drone is visible in gazebo.
> In terminal 3,
> 
> ```
> rosservice call /mavros/set_mode "base_mode: 0 custom_mode: 'OFFBOARD'"
> rosservice call /mavros/cmd/arming "value: true"
> ```
thanks for your reply, I did like you said before, it is exactly issue.
![image](https://user-images.githubusercontent.com/26245593/61377064-c385b980-a8a2-11e9-97f0-8257cd137380.png)

**Comment by harishanand95** [comment]
These errors are fine. You should still be able to get it flying. Could you show me the errors from terminal 2 and terminal 3?

**Comment by yiyongrobotics** [comment]
> These errors are fine. You should still be able to get it flying. Could you show me the errors from terminal 2 and terminal 3?


thanks, terminal 2 and terminal 3 
![image](https://user-images.githubusercontent.com/26245593/61440440-cf758800-a943-11e9-9fda-75d62819287a.png)

![image](https://user-images.githubusercontent.com/26245593/61440408-b79e0400-a943-11e9-8195-6438529812b6.png)

**Comment by darknight-007** [comment]
Are you trying to run this on the OpenUAV testbed or standard PX4 SITL? Sorry, I am just tuning into the conversation.
